Framework/Core/include/Framework/DataTypes.h

```diff
@@ -28,4 +27,16 @@ enum TrackFlagsRun2Enum {
 };
 } // namespace o2::aod::track
 
+namespace o2::aod::fwdtrack
+{
+enum ForwardTrackTypeEnum : uint8_t {
+  MUONStandalone = 0,
+  MFTStandalone,
+  GlobalMuonTrack,
```

Comment:
GlobalMuonTrackOtherMatch

Reply:
You removed the "BestMatch" from the GlobalMuonTrack on purpose?

Reply:
Even if a detail, I would start the enum with the most used one, which is GlobalMuonTrack(BestMatch), right?

Reply:
Moving GlobalMuonTrack to start